gurubu-client/src/app/components/page/testimonial.tsx

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image";
+
+const Testimonial = () => {
+  return (
+    <section className="testimonial-section">
+      <div className="testimonial">
+        <Image
+          width={123}
+          height={27}
```

Comment:
we can enlarge our logo like in figma
```suggestion
          width={140}
          height={40}
```